integrations/salesforce/src/misc/utils/schemaUtils.ts

```diff
@@ -0,0 +1,27 @@
+import { z, ZodObject, ZodRawShape, ZodTypeAny } from "@botpress/sdk";
+
+export const getSearchOutputSchema = (result: ZodObject<ZodRawShape>) => {
```

Comment:
would "record" be more aligned instead of "result"?

Reply:
Agree. Implemented